src/validateRegisterForm.test.js

```diff
@@ -26,5 +26,57 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
 
-  // write more tests here
+  it(`should return error for valid email and password less 
+  than 8 characters`, () => {
+    const invalidPassword = validateRegisterForm('test@mail.com', 'P@ss1');
+
+    expect(invalidPassword.code).toBe(422);
+    expect(invalidPassword.message).toBe('Password is invalid.');
+  });
+
+  it(`should return error for valid email and password greater
+   than 16 characters`, () => {
+    const invalidPassword = validateRegisterForm('test@mail.com',
+      'P@ssword123456789!');
+
+    expect(invalidPassword.code).toBe(422);
+    expect(invalidPassword.message).toBe('Password is invalid.');
+  });
+
+  it(`should return error for invalid email without '@' symbol`, () => {
+    const invalidEmail = validateRegisterForm('testmail.com', 'P@ssword1!');
```

Comment:
If entered password and email are invalid object contains status code 500 and message Password and email are invalid. This requirement isn't covered